remme/clients/pub_key.py

```diff
@@ -29,31 +31,33 @@
 from cryptography.hazmat.backends import default_backend
 
 
-class CertificateClient(BasicClient):
+class PubKeyClient(BasicClient):
     def __init__(self, test_helper=None):
-        super().__init__(CertificateHandler, test_helper=test_helper)
+        super().__init__(PubKeyHandler, test_helper=test_helper)
 
     @classmethod
-    def get_new_certificate_payload(self, certificate_raw, signature_rem, signature_crt, cert_signer_public_key):
-        payload = NewCertificatePayload()
-        payload.certificate_raw = certificate_raw
-        payload.signature_rem = signature_rem
-        payload.signature_crt = signature_crt
-        if cert_signer_public_key:
-            payload.cert_signer_public_key = cert_signer_public_key
+    def get_new_pub_key_payload(self, public_key, entity_hash, entity_hash_signature, valid_from, valid_to):
+        payload = NewPubKeyPayload()
+        payload.public_key = public_key
+        payload.public_key_type = NewPubKeyPayload.RSA
+        payload.entity_type = NewPubKeyPayload.PERSONAL
```

Comment:
let's add this enum values to arguments.
can we set default values for them, so function consumer will have options to specify them or use defaults ?